build_service/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:20.04
+
+RUN apt-get -y  update && apt-get -y upgrade
+
+RUN bash -c 'echo "deb http://dk.archive.ubuntu.com/ubuntu/ xenial main" >> /etc/apt/sources.list'
+RUN bash -c 'echo "deb http://dk.archive.ubuntu.com/ubuntu/ xenial universe" >> /etc/apt/sources.list'
+RUN apt update
+
+RUN apt -y install g++-4.9
+RUN apt -y install g++-4.9-multilib
+RUN bash -c 'head -n -2 /etc/apt/sources.list > intermediate_file;mv -f intermediate_file /etc/apt/sources.list'
```

Comment:
should there be another 'apt update' after this change to sources.list?

Reply:
I have tested this out-of-box build and it works as expected without `apt update` after change to sources.list

Reply:
It is a good practice to run 'apt-get update' after we change the source file. This makes the apt repository and source file in sync. That way, we can also ensure that all subsequent apt-get calls don't use the source files "http://dk.archive.ubuntu.com/ubuntu/ xenial universe" for package listing.